src/_nebari/subcommands/deploy.py

```diff
@@ -84,6 +84,11 @@ def deploy(
                     stages.remove(stage)
             rich.print("Skipping remote state provision")
 
+        # Digital Ocean support deprecation warning -- Nebari 2024.7.1
+        if config.provider == "do" and not disable_prompt:
+            msg = "Digital Ocean support is currently being deprecated and will be removed in a future release. Would you like to continue? [y/N]"
```

Comment:
Do you need to have the `[y/N]` string here? I think `typer` adds that by default so it will probably get duplicated.

Reply:
nice catch, I will double check that